eval/measure_judged.py

```diff
@@ -54,6 +56,10 @@ def load_run(path: str) -> Dict[str, List[str]]:
     return sorted_run
 
 
+def print_divider():
+    print('\n--------------------- \n')
```

Comment:
nbd, but is the extra space at the end intentional?

Reply:
Removed all loading qrels logging